tests/test_events.py

```diff
@@ -216,6 +216,10 @@ def test_current_meeting_no_streaming_event_late_start(event, mocker):
     }
     late_current_meeting = event.build(**crenshaw_meeting_info)
 
+    # Remove the broadcast since it hasn't been started. If it were kept,
+    # it would be filtered out by potentially_current_meetings
+    late_current_meeting.broadcast.get().delete()
```

Comment:
Great catch! It'd be great if we could pass in a keyword argument to event.build() that allows us configure whether to [build the event broadcast](https://github.com/Metro-Records/la-metro-councilmatic/blob/5931c6debd49c8bd4c7286f20866f261c1520bc5/tests/conftest.py#L148) regardless of event time. Want to open an issue for the improvement?

Reply:
Sounds good! I've got that issue opened and self-assigned.